### PR TITLE
fix(documentation,internet-header): Clarify and fix internet-header getting started doc

### PR DESCRIPTION
**Pull request by imagoiq**
This is coming from my analysis on https://github.com/swisspost/design-system/issues/1526 and https://github.com/swisspost/design-system/issues/1721. Please let me know if something doesn't make sense.

### CODE REVIEW

File: packages/documentation/src/stories/internet-header/getting-started.stories.mdx [hunk 1]
```diff
@@ -9,13 +9,22 @@ import { BADGE } from '@geometricpanda/storybook-addon-badges';
...
 The header for client-facing applications.
 
-- [Installation](#installation)
-- [Configuration](#configuration)
-- [Help](#help)
+<ul>
+  <li><a href="#installation" target="_self">Installation</a></li>
+  <li><a href="#configuration" target="_self">Configuration</a></li>
+  <li><a href="#help" target="_self">Help</a></li>
+</ul>
```
Comment (imagoiq) [inline packages/documentation/src/stories/internet-header/getting-started.stories.mdx:16]: There were redirecting to the iframe.html page. I'll do another to fix them across all doc. Storybook's issue: https://github.com/storybookjs/storybook/issues/15934

File: packages/documentation/src/stories/internet-header/getting-started.stories.mdx [hunk 2]
```diff
@@ -40,46 +42,63 @@ defineCustomElements();
...
+    <swisspost-internet-header project="your-project-id"></swisspost-internet-header>
+  </body>
+  <script type="module">
+    import { defineCustomElements } from '{cdnUrl}@{version}/loader/index.es2017.js';
+
+    defineCustomElements();
+  </script>
+</html>
+```
+#### Bare component
```
Comment (imagoiq) [inline packages/documentation/src/stories/internet-header/getting-started.stories.mdx:86]: Probably a good place to use the future tabs pattern.

Comment (gfellerph) [reply packages/documentation/src/stories/internet-header/getting-started.stories.mdx:86]: I opened a ticket for it https://github.com/swisspost/design-system/issues/1818

**Review by gfellerph — CHANGES_REQUESTED** [review]
Merge after the switch

**Review by gfellerph — APPROVED** [review]

File: .changeset/few-ducks-design.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+'@swisspost/internet-header': patch
+'@swisspost/design-system-documentation': patch
+---
+
+Clarify and fix which internet-header versions are available and how to use them.
```
Comment (imagoiq) [inline .changeset/few-ducks-design.md:6]:
```suggestion
Clarified and fixed which internet-header versions are available and how to use them.
```

**Pull request merged**